users.json

```diff
@@ -0,0 +1,1423 @@
+{
```

Comment:
is the SPID ID globally unique? I thought it was unique in the scope of a single IdP